internal/task/read_request.cpp

```diff
@@ -65,19 +49,13 @@ Result<IOTaskResult, std::string> ReadRequest::execute() {
         // message-body
         const size_t &body_size = content_length.unwrap();
         char *buf = new char[body_size];
-        const Result<size_t, std::string> read_body_result = reader_->read(buf, body_size);
-        if (read_body_result.isErr()) {
-            return Err(read_body_result.unwrapErr());
-        }
+        TRY(reader_->read(buf, body_size));
         maybe_body = Some(std::string(buf, body_size));
         delete[] buf;
     }
 
-    Result<Request, std::string> parse_result = RequestParser::parseRequest(request_line, headers_, maybe_body.unwrapOr(""));
-    if (parse_result.isErr()) {
-        return Err(parse_result.unwrapErr());
-    }
-    ctx_->setRequest(parse_result.unwrap());
+    Request parse_result = TRY(RequestParser::parseRequest(request_line, headers_, maybe_body.unwrapOr("")));
+    ctx_->setRequest(parse_result);
```

Comment:
命名そのままなので、変えておきたいですね。

```suggestion
    Request parsed_request = TRY(RequestParser::parseRequest(request_line, headers_, maybe_body.unwrapOr("")));
    ctx_->setRequest(parsed_request);
```